src/algorithm/broadphase/brute_force.rs

```diff
@@ -0,0 +1,126 @@
+use super::*;
+use prelude::*;
+
+/// Broad phase collision detection brute force implementation.
```

Comment:
perhaps, `broad_phase.rs` would be better?

Reply:
Agreed.